Doc/tutorial/errors.rst

```diff
@@ -20,12 +20,12 @@ complaint you get while you are still learning Python::
    >>> while True print('Hello world')
      File "<stdin>", line 1
        while True print('Hello world')
-                      ^
+                  ^^^^^
    SyntaxError: invalid syntax
 
-The parser repeats the offending line and displays a little 'arrow' pointing at
-the earliest point in the line where the error was detected.  The error is
-caused by (or at least detected at) the token *preceding* the arrow: in the
+The parser repeats the offending line and displays little 'arrow's pointing
+at the token in the line where the error was detected.  The error may be
+caused by the absence of a token *before* the indicated token.  In the
```

Comment:
Nit: double space between `token.` and `In` or at least it appears to be so in the GH, but when I copy-paste this, double space is transformed in a single space
![image](https://github.com/python/cpython/assets/39376984/3b6db6c2-6d6c-4cc3-a081-6fd6670381e1)
![image](https://github.com/python/cpython/assets/39376984/b8a2c72e-3298-431a-8c5c-de9a386acbc1)

Reply:
The .rst files are often (generally?) double-spaced after sentence ending periods.  [Devguide](https://devguide.python.org/documentation/style-guide/#use-of-whitespace) says "A sentence-ending period may be followed by one or two spaces; while reST ignores the second space, it is customarily put in by some users, for example to aid Emacs’ auto-fill mode."  Sphinx then does whatever it does when generating .html files and they are displayed however they are in a particular displayer.

Reply:
Thanks, somehow missed that part :)